tsinfer/formats.py

```diff
@@ -933,16 +945,17 @@ class SampleData(DataContainer):
     """
 
     FORMAT_NAME = "tsinfer-sample-data"
-    FORMAT_VERSION = (5, 0)
+    FORMAT_VERSION = (5, 1)
```

Comment:
I don't think this is a minor version bump - don't we need to do any upgrade to handle missing ``time_units`` values on older files?